script.js

```diff
@@ -290,23 +297,26 @@ function derive_power(expr, variable) {
     return part2;
   }
   if (is_number(f) && is_number(g) !== true) {
-    const part1 = make_product(
+    /*const part1 = make_product(
```

Comment:
if it is not needed, you can simply remove it.

Reply:
ok